xssp_rest/tasks.py

```diff
@@ -94,6 +94,14 @@ def mkhssp_from_sequence(sequence, output_format):
 
     If present, the input FASTA description line is used.
     """
+
+    if output_format == 'hssp_stockholm':
```

Comment:
Why `hssp_stockholm`? The user interface in the browser has these options. Are you asserting that `hssp_stockholm` and `hg_hssp` are the same?